spec/services/playoff_stage_service_spec.rb

```diff
@@ -51,17 +51,22 @@
     end
   end
 
-  describe '#generate_playoffs' do
+  describe 'generates playoff stages for' do
```

Comment:
See comment above.
This shows a good example of why it's a bad idea testing those functions like this.